tables.js

```diff
@@ -0,0 +1,242 @@
+const AWS = require('aws-sdk');
+const config = require('./config');
+
+AWS.config.update(config);
+
+// function createLocations() {
```

Comment:
Understand: Why are these functions commented out and not deleted or uncommented?